writing/milestone-0/OVERVIEW.md

```diff
@@ -0,0 +1,80 @@
+## Overview
+Substrate is a new kind of computer console specialized for running AI at home and work
+
+The console is paired with an open source operating system and suite of applications. Our goal is to make both hardware and software radically accessible —  to teach the world not just how to use the AI and appliance, but how they can make (and re-make!) it themselves.
+
+Our first iteration of the console will be a kit that people can build in their own home or office.
+
+Today, upgrading a computer usually means replacing components — but our unique approach is to design modular systems users can upgrade through expansion. 
+
+In practice this means a single initial machine with cables that can be plugged into one or more "expansion boxes". People can invest in expanding these systems over many years. With each expansion, the system can assist in more activities and bring more smarts to each of them.
+
+We have plans for single-, double-, and quad-accelerator expansion boxes that can be plugged into a variety of machines. The plans will be available in early 2024. The double reuses all the parts of the single, and the quad reuses all the parts of the single and double.
+Who It's For
+Initially, this project will serve hackers who want to build and collaborate with one another on local and intelligent systems. 
+
+### Who is it for? What are some sample situations?
+
+A1. Hackers (working on an open source software/hardware project)
+
+A2. Students (in grad school doing research, with international and multilingual collaborators)
+
+A3. Small and mid-sized businesses (a regional distributor)
+
+
+### What It Is
+
+#### What you get in the mail:
+- Substrate System 0 is an easy-to-assemble AI computer kit for home and work.
+
+#### Once built, it runs:
+- SubstrateOS, an operating system that boots from a USB drive and schedules AI models on internal and external consumer GPUs.
+
+#### Once booted, it can:
+- M0.1. Join in on real-world conversations via your other devices. [A1] [A2] [A3]
+    A1. Planning meeting
+    A2. Lecture
+    A3. Sales meeting, customer call
+
+- M0.2. Take meeting and lecture notes so you can pay full attention to the speakers. [Liv] [A1] [A2] [A3]
+    A1. Meeting minutes, next steps
+    A2. Lecture notes
+    A3. ?
+
+- M0.3. Identify and link to more info on people, events, and cited documents. [Liv] [A1] [A2]
+    A1. ?
+    A2. Important dates for the course; references to other talks, textbooks, papers, textbooks
+
+- M0.4. Keep an at-hand, ever-present record of conversation, tasks, and specific contributions. This captures good ideas as they happen, foster new connections between them, and helps maintain understanding between collaborators. [Matt] [Liv] [A1] [A2] [A3]
+    A1. Feature suggestions, observations, 
+    A2. ?
+    A3. ?
```

Comment:
A3. Map, track, and attribute ideas as they emerge and evolve in employee-employee correspondence. I spoke about how bad companies are at this with researcher Lauren Klein: https://complexity.simplecast.com/episodes/70 ...one big takeaway from that conversation is that we could be using techniques from the digital humanities to dramatically improve the circumstances for innovation in organizations, as well as happiness in the workplace. It's not just about encouraging the production of good ideas but also ensuring people are fairly recognized and compensated. This could neutralize one of the most widespread forms of organizational pathology, which is people stealing each other's credit and/or becoming obsessive about preventing this to a degree that harms collaboration.

Reply:
which techniques from the digital humanities?

Reply:
Topic modeling.  This kind of visualization is something I would salivate over as a near-future Substrate feature for this specific use case:
https://medium.com/@power.up1163/visualizing-topic-models-with-topicwizard-ee5b4428405e

Reply:
Can you give a short example in screenplay/narrative form that shows how this might work in any of our specific settings?

Reply:
A1. At a glance, members of a software development team can see a network map of updates proposed by collaborators without having to scan a huge linear document and check for modifications one by one. Being able to see the "overhead" view of the project draws talent to the problems that need fixing quicker and more efficiently and creates "attention basins" by comparing different-but-related codebases that can attract talent _from other projects_ into areas where they may be able to make significant updates.
A2. Grad students in a biology lab manage to successfully petition to reorder their names in a major scientific publication based on Substrate's records of lab work and meetings. Consequently, the students who contributed the most to that paper are awarded research opportunities for which they might otherwise have been passed over. (Note: citation bias is a HUGE issue — https://physics.aps.org/articles/v16/15 — and addressing this issue at the headwaters reduces even more pernicious problems downstream.)
A3. It's time for annual bonuses and thanks to Substrate tracking workplace meetings it was easy for leadership to run a topic model on the best ideas that have come out of working groups this year. As it happens, three of the best ideas came from low-level staff, who were granted season-changing bonuses for being unsung sources of major innovations. (And one of the best ideas had been claimed by someone else, which resolved some major resentment in the office.  Justice!)